test/e2e/operator_test.go

```diff
@@ -144,9 +149,11 @@ func TestVersionReporting(t *testing.T) {
 		}
 	}()
 
+	name := types.NamespacedName{Name: operatorcontroller.DNSOperatorName}
 	err = wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
 		co := &configv1.ClusterOperator{}
-		if err := cl.Get(context.TODO(), types.NamespacedName{Name: operatorcontroller.DNSOperatorName}, co); err != nil {
+		if err := cl.Get(context.TODO(), name, co); err != nil {
+			t.Logf("failed to get deployment %s: %v", name, err)
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s: %v", namespacedName.Name, err)`